ios/departureWidget/Views/ChipView.swift

```diff
@@ -1,15 +1,17 @@
 import SwiftUI
 
 struct ChipView: View {
-    let label: String
+    let departure: DepartureLinkLabel
 
     var body: some View {
-        Text(label)
-            .lineLimit(1)
-            .fixedSize()
-            .padding(8)
-            .background(Color("TimeTileBackgroundColor"))
-            .cornerRadius(8)
-            .font(DefaultFonts.bold)
+        Link(destination: URL(string: departure.link)!) {
```

Comment:
Would be nice to unwrap this `URL(string: departure.link)!` instead of force unwrap.

Reply:
This works?